### PR TITLE
Improve enrich image

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Tiny improvement to have tags with proper `pr-sha` tags.

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 255bc00dbe875d7ff64984570a6bc74ed032e1ad with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55793/255bc00dbe875d7ff64984570a6bc74ed032e1ad/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6564579970">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/55793/255bc00dbe875d7ff64984570a6bc74ed032e1ad/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Comment by Felixoid** [comment]
https://github.com/ClickHouse/ClickHouse/actions/runs/6565040826/job/17832955748?pr=55793#step:6:101

it works

File: tests/ci/docker_manifests_merge.py [hunk 1]
```diff
@@ -209,7 +209,10 @@ def enrich_images(changed_images: Dict[str, str]) -> None:
         """
 
     batch_count = 0
-    ch_helper = ClickHouseHelper()
+    # We use always publibly available DB here intentionally
```
Comment (mkaynov) [inline tests/ci/docker_manifests_merge.py:212]: typo

**Review by maxknv — APPROVED** [review]

**Comment by antaljanosbenjamin** [comment]
Thanks @Felixoid for the very nice fix!

**Pull request merged**